R/datasets.R

```diff
@@ -154,18 +154,20 @@ ImmGenData <- function(){
 #' @return A \linkS4class{SummarizedExperiment} object with a \code{"normcounts"} assay
 #' containing the normalized expression values and cell type labels in the \code{\link{colData}}.
 #'
-#' @author Friederike Duendar
+#' @author Friederike Dündar
 #' 
 #' @references
 #' Benayoun B et al. (2019).
 #' Remodeling of epigenome and transcriptome landscapes with aging in mice reveals widespread induction of inflammatory responses.
 #' \emph{Genome Research}. doi: 10.1101/gr.240093.118
 #' 
+#' Code at: \code{https://github.com/BenayounLaboratory/Mouse_Aging_Epigenomics_2018/tree/master/FigureS7_CIBERSORT/RNAseq_datasets_for_Deconvolution/2017-01-18}
+#' 
 #' @examples
-#' ref.se <- MouseBulkData()
+#' ref.se <- MouseData()
 #' 
 #' @export
-MouseBulkData <- function(){
+MouseData <- function(){
```

Comment:
Sort of vague. Perhaps we could call it `MouseSortedCellData`?

Reply:
At this point, I'm not sure how much scRNA-seq (of tissues) is in there. Maybe `MouseTissueData`?

Reply:
Why tissue? Aren't they sorted cell populations? Tissue would make sense if e.g., you're mushing a whole liver and sequencing that as a single bulk sample.

Reply:
The table that Berenice sent me that supposedly contained the sample IDs for the `mouse.rnaseq` object held numerous scRNA-seq data sets, probably from Tabula muris, with different degrees of granularity. There's only about 290 bulk RNA-seq samples in that table, we have more than 300 samples in our object, so I need to understand which samples ultimately made it into the one object we're pulling from Dvir. Anyway, it made me think that Dvir's original name`mouse.rnaseq` is probably as precise as he could be.

Reply:
`MouseCellData`, perhaps?

Reply:
`MouseRNAseqData` ? `MouseRefData`?

Reply:
You don't like `Cells`? I mean, that's pretty much what they are.

Reply:
`Cells` is too vague for me
`RNAseq` is  a bit more informative because it implies that it's not microarray data

Reply:
plus it is reflecting the name of the original rda